dev/com.ibm.ws.springboot.fat20.concurrency.app/src/main/java/com/example/demo/ConcurrencyApplicationConfig.java

```diff
@@ -0,0 +1,70 @@
+package com.example.demo;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import javax.naming.NamingException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.DefaultManagedTaskExecutor;
+import org.springframework.scheduling.concurrent.DefaultManagedTaskScheduler;
+import org.springframework.stereotype.Service;
+
+@Configuration(proxyBeanMethods = false)
+@EnableAsync
+public class ConcurrencyApplicationConfig {
+	
+	private final static Logger logger = LoggerFactory.getLogger(ConcurrencyApplicationConfig.class);
+   
+    @Bean
+    public DefaultManagedTaskScheduler defaultManagedTaskScheduler() throws InterruptedException {
+    	DefaultManagedTaskScheduler scheduler = new DefaultManagedTaskScheduler();
+    	scheduler.setConcurrentExecutor(taskExecutor());
+        return scheduler;
+    }
+    
+    @Bean
+    public DefaultManagedTaskExecutor taskExecutor() throws InterruptedException {
+    	DefaultManagedTaskExecutor executor = new DefaultManagedTaskExecutor();
+        return executor;
+    }
```

Comment:
I suggest these bean methods for creating the executors get moved to the `ConcurrencyApplication` class.

I also suggest this class be renamed to `ConcurrencyTasks`

Reply:
Beans have been moved and the class has been renamed.